test-project/Assets/Generated/Source/improbable/gdk/tests/blittabletypes/BlittableComponentGameObjectComponentDispatcher.cs

```diff
@@ -300,18 +301,24 @@ public override void InvokeOnCommandResponseCallbacks(Dictionary<Unity.Entities.
                     for (var i = 0; i < entities.Length; i++)
                     {
                         var injectableStore = entityToInjectableStore[entities[i]];
-                        if (!injectableStore.TryGetInjectablesForComponent(commandResponseHandlerInjectableId, out var commandResponseHandlers))
-                        {
-                            continue;
-                        }
+                        bool thereIsResponseHandler =
+                            injectableStore.TryGetInjectablesForComponent(commandResponseHandlerInjectableId,
+                                out var commandResponseHandlers);
 
                         var commandResponseList = commandResponseLists[i];
-                        foreach (Requirable.CommandResponseHandler commandResponseHandler in commandResponseHandlers)
+                        foreach (var commandResponse in commandResponseList.Responses)
                         {
-                            foreach (var commandResponse in commandResponseList.Responses)
+                            if (thereIsResponseHandler)
```

Comment:
Just check if `commandResponseHandlers` is `null` and get rid of the bool?

Reply:
sure